Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml

```diff
@@ -92,6 +92,18 @@
   components:
   - type: Foldable
     folded: true
+  - type: Sprite
+    layers:
+      - state: bag
+        map: [ "unfoldedLayer" ]
+        visible: false
```

Comment:
Given this a child of `BodyBag_Container` changing the `unfoldedLayer` to not visible and `folderLayer` to visible should suffice.

I think the real problem here is the logic at https://github.com/space-wizards/space-station-14/blob/44fb8a9e2da5cf8f9a2cd939a7b563bc4360ddb7/Content.Shared/Foldable/SharedFoldableSystem.cs#L38 though : setting `Foldable` to true should make the foldable system update the sprite.

Reply:
I actually did notice the greater issue with the foldable system when I looked into this issue, but I wasn't confident enough to make that change since this is both my first time fiddling with SS14 (coming from some minor SS13 stuff I did years ago) and I'm not entirely sure where else FoldableComponent is used to make sure I don't break other foldable items. I'll look into fixing that logic in a separate PR.